tests/workarounds/wrl/wrl/implements.h

```diff
@@ -2153,7 +2153,10 @@ class RuntimeClass :
     RuntimeClass(const RuntimeClass&);
     RuntimeClass& operator=(const RuntimeClass&);
 protected:
-    HRESULT CustomQueryInterface(REFIID /*riid*/, _Outptr_result_nullonfailure_ void** /*ppvObject*/, _Out_ bool *handled)
+    HRESULT CustomQueryInterface(

```

Comment:
IMO don't bother with the tests at all. If you _do_ fix something here, you should apply the same fix to the same file in the OS repo since these files are intended to be temporary and deleted eventually.